provider/bigquery.go

```diff
@@ -43,65 +44,31 @@ const (
 )
 
 type BQOfflineStoreConfig struct {
```

Comment:
Rename and get rid of serialized and just use the provider_config.BigQueryConfig